### DISCUSSION TITLE
Seriously great package and documentation

### DISCUSSION THREAD

**Discussion by zehawki** [anymail/django-anymail#265] [General]
Just wanted to drop in [another appreciation](https://github.com/anymail/django-anymail/issues/194) that I'm blown away by how good the documentation on this package is and how well it works. Thank you!

After 4 years of using SES, I'm looking to shift away from SES to Postmark. It took me 30 secs to make the switch and send the first mail out successfully. So well detailed here https://anymail.dev/en/stable/esps/postmark/

**Comment by medmunds** [comment]
Thanks, and glad your changeover is going smoothly.

This reminds me, I've been meaning to add some docs on "how to switch ESPs". It would cover things like:

* Grep your code for `esp_extra` and `esp_event`, which will have ESP-specific things that need updates
* If you're using `{OLDESP}_SEND_DEFAULTS` in your Anymail settings, those will have to be converted
* If you're using ESP stored templates… well, that's probably just going to be a lot of manual conversion, sorry
* If possible, export the blocked recipients list (spam/bounce addresses) from your old ESP and import it into your new ESP (not all ESPs support this)
* Don't forget to update your SPF, DKIM, and DMARC records
* Etc. 

(If you run into any other issues switching, maybe you could add a note here for future users.)
